lib/support/windows.js

```diff
@@ -222,17 +222,23 @@ const WindowsSupport = {
     });
   },
 
-  runKite() {
+  runKite(openCopilot) {
     return this.isKiteRunning()
     .catch(() => {
       var env = Object.create(process.env);
-      env.KITE_SKIP_ONBOARDING = '1';
       delete env["ELECTRON_RUN_AS_NODE"];
+      if (!openCopilot){
+        env.KITE_SKIP_ONBOARDING = '1';
```

Comment:
interesting: this doesn't exactly match the old logic, but it looks like it probably doesn't matter based on how `kited` is currently consuming this.